include/drivers/sensor.h

```diff
@@ -174,6 +174,15 @@ enum sensor_channel {
 	/** Desired charging current in mA */
 	SENSOR_CHAN_GAUGE_DESIRED_CHARGING_CURRENT,
 
+	/** Channel 0, raw data */
+	SENSOR_CHAN_RAW_CH0,
+	/** Channel 1, raw data */
+	SENSOR_CHAN_RAW_CH1,
+	/** Channel 2, raw data */
+	SENSOR_CHAN_RAW_CH2,
+	/** Channel 3, raw data */
+	SENSOR_CHAN_RAW_CH3,
+
```

Comment:
I'm not sure this makes sense to add here. @MaureenHelm ?

Reply:
I was also not happy about that, but I don't think the current sensor API is able to read multiple channels of the same type from a single device? Would be good to be able to pass something like a channel number to the channel get function e.g. `sensor_channel_get(dev, SENSOR_CHAN_RAW, ch_number, buffer);`. Is there a plan to add a functionality like that?

Reply:
This is a decision for the sensor maintainers; I can't comment on this one.

Reply:
I'm not a fan of adding channels without a type or standard unit. What does the quantity represent?

Reply:
It can be represented as Capacitance or Frequency. I can add those conversion. Should I leave the Channel 0-3  numbering at the end of each type?

Reply:
Added the conversions from raw to Capacitance and Freqzency.